flake.nix

```diff
@@ -25,7 +25,7 @@
     in
     {
       lib = lib.extend (final: prev: {
-        nixosSystem = { modules, ... } @ args:
+        nixosSystem = final.makeOverridable ({ modules, ... } @ args:
```

Comment:
Do we really need more and more complexity in the flake.nix? Shouldn't we have everything you need in nixpkgs already? I guess your change is reasonable but it just made me look at this file for the first time.

The whole complexity below is probably already present elsewhere in the repo. Is there a reason for not using that?

Reply:
I don't feel comfortable arguing for why flake.nix looks like it does, but my guess it'll be put elsewhere when flakes are the standard